### PR TITLE
NOISSUE - Return certs on bootstrap view response

### PR DESCRIPTION
**Pull request by SammyOina**
### What does this do?
- Retrieves Client_Cert, and CA_Cert when a view request is made on the bootstrap service.
- Returns certs on updateCert
- Use named parameters on queries

### Which issue(s) does this PR fix/relate to?
No issue

### Did you document any new/modified functionality?
I have added modified parameters to openapi spec

### CODE REVIEW

File: bootstrap/postgres/configs.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ func (cr configRepository) Save(ctx context.Context, cfg bootstrap.Config, chsCo
 }
 
 func (cr configRepository) RetrieveByID(ctx context.Context, owner, id string) (bootstrap.Config, error) {
-	q := `SELECT mainflux_thing, mainflux_key, external_id, external_key, name, content, state
+	q := `SELECT mainflux_thing, mainflux_key, external_id, external_key, name, content, state, client_cert, client_key, ca_cert
```
Comment (dborovcanin) [inline bootstrap/postgres/configs.go]: We should probably not return the client key.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

File: bootstrap/postgres/configs.go (hunk 1 above)
Comment (SammyOina) [reply bootstrap/postgres/configs.go]: ok, I'll leave to be returned on updateCert only

**Review by dborovcanin — APPROVED** [review]

File: provision/service.go [hunk 2]
```diff
@@ -226,7 +226,8 @@ func (ps *provisionService) Provision(token, name, externalID, externalKey strin
 			res.CACert = ""
 
 			if needsBootstrap(thing) {
-				if err = ps.sdk.UpdateBootstrapCerts(bsConfig.MFThing, cert.ClientCert, cert.ClientKey, "", token); err != nil {
+				_, err = ps.sdk.UpdateBootstrapCerts(bsConfig.MFThing, cert.ClientCert, cert.ClientKey, "", token)
+				if err != nil {
```
Comment (dborovcanin) [inline provision/service.go]: Inline.

File: bootstrap/service_test.go [hunk 3]
```diff
@@ -288,32 +289,50 @@ func TestUpdateCert(t *testing.T) {
...
+			expectedConfig: bootstrap.Config{},
+			err:            errors.ErrAuthentication,
 		},
 	}
 
 	for _, tc := range cases {
-		err := svc.UpdateCert(context.Background(), tc.token, tc.thingKey, tc.clientCert, tc.clientKey, tc.caCert)
+		cfg, err := svc.UpdateCert(context.Background(), tc.token, tc.thingKey, tc.clientCert, tc.clientKey, tc.caCert)
 		assert.True(t, errors.Contains(err, tc.err), fmt.Sprintf("%s: expected %s got %s\n", tc.desc, tc.err, err))
+		if err == nil {
```
Comment (dborovcanin) [inline bootstrap/service_test.go]: Is this check needed? In case of an error, the config will be empty.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

**Review by dborovcanin — DISMISSED** [review]

**Review by drasko — DISMISSED** [review]
LGTM

**Review by dborovcanin — DISMISSED** [review]

**Review by dborovcanin — APPROVED** [review]

**Review by dborovcanin — CHANGES_REQUESTED** [review]
Please fix CI remarks.

**Review by dborovcanin — DISMISSED** [review]

File: bootstrap/configs.go [hunk 4]
```diff
@@ -16,13 +16,13 @@ import (
...
-	MFThing     string    `json:"mainflux_thing"`
+	ThingID     string    `json:"thing_id"`
 	Owner       string    `json:"owner,omitempty"`
 	Name        string    `json:"name,omitempty"`
 	ClientCert  string    `json:"client_cert,omitempty"`
 	ClientKey   string    `json:"client_key,omitempty"`
 	CACert      string    `json:"ca_cert,omitempty"`
-	MFKey       string    `json:"mainflux_key"`
+	ThingKey    string    `json:"thing_key"`
 	MFChannels  []Channel `json:"mainflux_channels,omitempty"`
```
Comment (arvindh123) [inline bootstrap/configs.go]: @SammyOina, Is there any reason behind for not changing `mainflux_channels` to `channels` ?

Comment (SammyOina) [reply bootstrap/configs.go]: yes, the create request channels is of type []string and for view request it is of type []Channel

File: bootstrap/api/responses.go [hunk 5]
```diff
@@ -67,14 +67,16 @@ type channelRes struct {
 }
 
 type viewRes struct {
-	MFThing     string          `json:"mainflux_id,omitempty"`
-	MFKey       string          `json:"mainflux_key,omitempty"`
+	ThingID     string          `json:"thing_id,omitempty"`
+	ThingKey    string          `json:"thing_key,omitempty"`
 	Channels    []channelRes    `json:"mainflux_channels,omitempty"`
 	ExternalID  string          `json:"external_id"`
```
Comment (arvindh123) [inline bootstrap/api/responses.go:73]: if `mainflux_id` can be changed to `thing_id`,  `mainflux_key` to `thing_key` , then `mainflux_channels` can be changed to `channels`

**Comment by arvindh123** [comment]
@SammyOina 
There is an other PR , which changes the flow of Bootstrap & Certs 
https://github.com/mainflux/mainflux/pull/1669
https://github.com/ultravioletrs/issues/issues/124

File: bootstrap/api/responses.go (hunk 5 above)
Comment (SammyOina) [reply bootstrap/api/responses.go:73]: @arvindh123 theres a clash between requests and responses, what do you suggest https://github.com/SammyOina/mainflux/blob/70f53c2f979dbe408d9b1ad7ba0ba99a72eb0c70/pkg/sdk/go/bootstrap.go#L30-L35

**Comment by SammyOina** [comment]
> @SammyOina There is an other PR , which changes the flow of Bootstrap & Certs #1669 [ultravioletrs/issues#124](https://github.com/ultravioletrs/issues/issues/124)

I see but I don't expect them to clash

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**